lib/pbench/server/database/models/datasets.py

```diff
@@ -360,12 +368,30 @@ class Dataset(Database.Base):
     PRIVATE_ACCESS = "private"
     ACCESS_KEYWORDS = [PUBLIC_ACCESS, PRIVATE_ACCESS]
 
+    # Generated unique ID for Dataset row
     id = Column(Integer, primary_key=True, autoincrement=True)
+
+    # Dataset name
+    name = Column(String(255), unique=True, nullable=False)
+
+    # ID of the owning user
     owner_id = Column(Integer, ForeignKey("users.id"), nullable=False)
+
+    # Indirect reference to the owning User record
     owner = relationship("User")
+
+    # Access policy for Dataset (public or private)
     access = Column(String(255), unique=False, nullable=False, default="private")
+
+    # Host name of the system that collected the data
+    #
+    # TODO: Do we want to kick this out of the Dataset record and into
+    # metadata? I'm considering collecting all of the `metadata.log` values
+    # into metadata for easy reference, and perhaps controller is just a part
+    # of that now? The main issue is that figuring out how to integrate the
+    # JSON Metadata table into a Dataset query (via a join or sub-select?) will
+    # be somewhat complicated, and I'd rather not do it now.
```

Comment:
Seems like this `TODO` would be better off in an issue.

Reply:
We should probably state that `controller` is no longer considered when determining tar ball uniqueness.

Reply:
Perhaps, rather than immediately as a GitHub Issue, this should first be captured as a Story in JIRA?